native/packages/react-native-bpk-component-button/readme.md

```diff
@@ -56,6 +56,58 @@ export default class App extends Component {
 }
 ```
 
+### With Theming
```

Comment:
I reckon this section is unnecessary as it's duplicated in the theming page. For each themeable component, jsut listing the props shoudl be enough.